spec/features/views_game_description_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+
+  feature 'user views the details of a game' do
+    let(:game) { FactoryGirl.create(:game) }
+    let(:game1) { FactoryGirl.create(:game1) }
+
+    scenario 'user views game details' do
+      sign_up
+      game
+      visit "/games"
+      click_link "#{game.name}"
+      expect(page).to have_content(game.name)
+      expect(page).to have_content(game.description)
+      expect(page).to have_content(game.min_players)
+      expect(page).to have_content(game.max_players)
+      expect(page).to have_content(game.playing_time)
+      expect(page).to have_content(game.complexity)
+    end
+
+    scenario 'user doesnt view other games details' do
+      sign_up
+      game1
+      game
+      visit "/games"
+      click_link "#{game.name}"
+      expect(page).not_to have_content(game1.name)
+      expect(page).not_to have_content(game1.description)
+      expect(page).not_to have_content(game1.min_players)
+      expect(page).not_to have_content(game1.max_players)
+      expect(page).not_to have_content(game1.playing_time)
+      expect(page).not_to have_content(game1.complexity)
+    end
+
+    scenario 'visitor views game details' do
+      game
+      visit "/games"
+      click_link "#{game.name}"
+      expect(page).to have_content(game.name)
+      expect(page).to have_content(game.description)
+      expect(page).to have_content(game.min_players)
+      expect(page).to have_content(game.max_players)
+      expect(page).to have_content(game.playing_time)
+      expect(page).to have_content(game.complexity)
+    end
```

Comment:
If the behavior of the page is the same for a guest and for an authenticated user you can probably just have a single test as a visitor.